tools/data/hvu/download_annotations.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
```

Comment:
Better to use `set -e` at the beginning. ref https://stackoverflow.com/questions/3474526/stop-on-first-error